crates/wysiwyg/src/dom/find_result.rs

```diff
@@ -12,15 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::dom::dom_handle::DomHandle;
+use crate::dom::range::DomLocation;
 
 #[derive(Clone, Debug, PartialEq)]
 pub enum FindResult {
```

Comment:
`FindResult` might not even be needed anymore, we could just return the list of locations. If it's empty, it's the same as `NotFound`. What do you think?